pkg/tfgen/generate_python.go

```diff
@@ -616,6 +626,13 @@ func (g *pythonGenerator) emitResourceFunc(mod *module, fun *resourceFunc) (stri
 	}
 	defer contract.IgnoreClose(w)
 
+	if fun.info.DeprecationMessage != "" {
+		w.Writefmtln("warnings.warn(")
+		w.Writefmtln("    \"%s\",", fun.info.DeprecationMessage)
+		w.Writefmtln("    DeprecationWarning")
+		w.Writefmtln(")")
```

Comment:
Any particular reason we are splitting this over 4 lines of generated code?  Seems it may as well just be one.

Reply:
This is just to ensure that PEP3 doesn't get tripped up when we have a longer message